.github/workflows/create-release-pr.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           node-version: '20'
 
       - name: Install
-        run: yarn install
+        run: yarn install --frozen-lockfile
```

Comment:
The problem is in the publish.yaml file.

Reply:
I added in both now